extensions/controllers/sandboxclaim_controller.go

```diff
@@ -250,8 +250,10 @@ func (r *SandboxClaimReconciler) createSandbox(ctx context.Context, claim *exten
 	logger.Info("creating sandbox from template", "template", template.Name)
 	sandbox := &v1alpha1.Sandbox{
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace: claim.Namespace,
-			Name:      claim.Name,
+			Namespace:   claim.Namespace,
+			Name:        claim.Name,
+			Labels:      claim.Labels,
+			Annotations: claim.Annotations,
 		},
 	}
```

Comment:
lgtm, not blocking but do you mind updating the `sandbox_claim.yaml` https://github.com/kubernetes-sigs/agent-sandbox/blob/main/extensions/examples/sandboxclaim.yaml to show that we can pass labels/annotations through the claim as well , thanks!